recipes/nanoeigenpy/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: nanoeigenpy
+  version: "0.1.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/Simple-Robotics/${{ name }}/releases/download/v${{ version }}/nanoeigenpy-${{ version }}.tar.gz
+  sha256: 06232d011a9c7f7eb72173bd39ff023b21f6462baa13309c39af13a4c4ba6942
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+    - cmake
+    - ninja
+    - git
+    - nanobind
```

Comment:
nanobind is not a build tool, is a library. So it's better to put it in host section.